### PR TITLE
Rename OMEdit's omc_config.h to omedit_config.h

### PR DESCRIPTION
**Pull request by mahge**
- This is to avoid conflicting names with `OMCompiler/omc_config.h`

### CODE REVIEW

**Review by adeas31 — APPROVED** [review]
Good! I did some cleanup of includes.

File: OMEdit/OMEditLIB/MainWindow.cpp [hunk 1]
```diff
@@ -75,7 +75,6 @@
 #include "Plotting/DiagramWindow.h"
 #include "Interfaces/InformationInterface.h"
 #include "Interfaces/ModelInterface.h"
-#include "omc_config.h"
 #include "Util/NetworkAccessManager.h"
 #include "Modeling/InstallLibraryDialog.h"
 #include "CrashReport/CrashReportDialog.h"
```
Comment (mahge) [inline OMEdit/OMEditLIB/MainWindow.cpp:80]: This one seems to be needed. The definition of HOST_SHORT macro is in `omedit_config.h`

Comment (adeas31) [reply OMEdit/OMEditLIB/MainWindow.cpp:81]: Yes. The windows and linux config files are not the same :)

Comment (mahge) [reply OMEdit/OMEditLIB/MainWindow.cpp:81]: We can fix that soon. Hopefully.

**Pull request merged**